src/components/notes/list.jsx

```diff
@@ -1,116 +1,75 @@
 import React from 'react'
 
 import { MainTitle } from 'cozy-ui/react/Text'
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableHeader,
+  TableCell
+} from 'cozy-ui/react/Table'
+import Empty from 'cozy-ui/react/Empty'
+import Stack from 'cozy-ui/react/Stack'
+import { translate } from 'cozy-ui/react/I18n'
+import { withBreakpoints } from 'cozy-ui/transpiled/react'
 
 import Add from './add'
-
-import HeaderMenu from '../header_menu.jsx'
-
-/*
-
-import Spinner from 'cozy-ui/react/Spinner'
-import { Link } from 'react-router-dom'
-
-import ListItemText from 'cozy-ui/react/ListItemText'
-import Button from 'cozy-ui/react/Button'
-import Icon from 'cozy-ui/react/Icon'
-
-import icon from '../../assets/icons/icon-note-32.svg'
-
-
-
-const titleWithDefault = () => {}
-
-const Item = props => (
-  <div className="note-item">
-    <Icon icon={icon} width={32} height={32} className="note-icon" />
-    <Link to={`/n/${props.note.id}`} className="note-link">
-      <ListItemText
-        primaryText={titleWithDefault(props.note)}
-        secondaryText="/Notes/2019/demo"
-      />
-    </Link>
+import icon from '../../assets/icons/icon_note_empty.svg'
```

Comment:
Out of scope for this PR I guess, but shouldn't we use absolute pathes that can be overriden with webpack aliases?

Reply:
Done thanks